packages/docs-yaml/src/helper/common.ts

```diff
@@ -0,0 +1,34 @@
+'use-strict';
+
+import { Uri, window, workspace } from 'vscode';
+import { reporter } from '../helper/telemetry';
+export const output = window.createOutputChannel('docs-yaml');
+
+/**
+ * Return repo name
+ * @param Uri
+ */
+export function getRepoName(workspacePath: Uri) {
+	// let repoName;
```

Comment:
Commented out code. No impact on functionality and we can clean this out in another PR if needed.